server/services/courseService.ts

```diff
@@ -36,6 +36,19 @@ export default class CourseService {
     return courseClient.destroyParticipation(courseParticipationId)
   }
 
+  async getAndPresentParticipationsByPrisonNumber(
```

Comment:
The naming of this is a bit out of sync with `getParticipationsByPerson` - they're both by prison number, but ones uses this in the name and one abstracts it to the person. Would be good to be consistent